modules/storage-persistent-storage-change-policy-overview.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//* microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="storage_persistent_storage_change_policy_overview_{context}"]
+= Reducing pod timeouts
+
+Certain volume characteristics can cause pod startup delays and timeouts. The following table lists some volume issues that can cause pod timeout with a possible solution.
+
+.Possible pod startup delay issues and solutions
+[cols="2",options="header"]
+|===
+|Issue
+|Solution
+
+|Volume with lots of files
+|Set `fsGroupChangePolicy` to stop recursively changing ownership and permissions to match the fsGroup specified in a pod’s `securityContext`. 
+
+|Using SELinux on large volumes
```

Comment:
So in both case fsgroup and selinux this is due to number of files not necessarily "large volumes". This can also be due to a backend being slow at updating file's metadata but let's not go into this here.